src/controllers/tasks/people.with.significant.control.controller.ts

```diff
@@ -23,13 +24,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
+const getTemplate = (isRle): string => {
+  if (isRle) {
+    return Templates.PEOPLE_WITH_SIGNIFICANT_CONTROL_RLE;
```

Comment:
as we can't unit test this new code and it can't be reached yet due to the 'false' I'd say don't commit the controller code,  just have the template in the PR.